effortless-chef-io/effortless-audit.md

```diff
@@ -182,7 +182,7 @@ With the release of `scaffolding-chef-inspec` version 0.16.0 (Linux) and version
 1. Apply the new change to your Chef Habitat config:
 
    ```bash
-   hab config apply <your profile service>.<your profile service group> $(date) <your config toml file>
+   hab config apply <my_profile_service>.<my_profile_service_group> $(date) <my_config.toml>
    ```
 
 1. Habitat will see a configuration change, automatically re-run your profile, and skip the control you specified in the `waiver.toml` file.
```

Comment:
@TheLunaticScripter Question! Is this `waiver.toml` file supposed to be a `waiver.yml` file? (See: https://www.inspec.io/docs/reference/waivers/) I think I might've mixed that up in my initial editing. NTS: I also made that happen in line 172.

Reply:
So this is a toml file because it is using the habitat config to then translate into an `inspec.yml` file. I think this needs to match the above `my_config.toml` change.

Reply:
@TheLunaticScripter Better?

Reply:
Yep I think that looks great.